src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -288,8 +290,16 @@ public void onTraversalStart() {
         final GenomeLocParser genomeLocParser = new GenomeLocParser(seqDictionary);
         intervalArgumentGenomeLocSortedSet = GenomeLocSortedSet.createSetFromList(genomeLocParser, IntervalUtils.genomeLocsFromLocatables(genomeLocParser, intervalArgumentCollection.getIntervals(seqDictionary)));
 
+        if (gqStateToIgnore != null) {
+            gqStatesToIgnore.add(gqStateToIgnore);
+            if (dropAboveGqThreshold) {
+                // TODO - Do we want to get rid of this option ("dropAboveGqThreshold") - never used and confuses things.
```

Comment:
Not sure I understand this comment, looks like it's being used [here](https://github.com/broadinstitute/gatk/blob/8245d48280cd5f9867ad4dfa70ef518465276dda/scripts/variantstore/wdl/GvsImportGenomes.wdl#L330)? Are you saying we *shouldn't* be using it / don't have use cases for it?

Reply:
Yeah this option is particularly nutty if we're dropping GQ 0, isn't it... 😬

Reply:
Hmm. It's true that it's settable there, but I don't think we ever use it?  @RoriCremer ?? Happy to leave it be.

Reply:
Yeah, if we were to need it, I almost think it should take a list of GQEnums.

Reply:
If you feel like ripping it out I'd say go for it, but I don't think it's required for this PR.

Reply:
agreed that it's not necessary for this pr. I do think that having a chat about cleaning up the enums and the params here in a tech debt conversation makes a lot of sense.

And it likely was settable in a WDL at some point when we were using multiple versions of reblocked data.
Recall that it's really hard to get large amounts of this data to scale test -- and before we could grumble and use AoU, we didn't even have that as an option, so sometimes multiple datasets would get cobbled together for scale testing, and we would need to deal with previously re-blocked data